### PR TITLE
pw-loopback: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/pw-loopback.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`pw-loopback`
+
+- Create a sink that can be selected in applications and that forwards the data to another sink:
+
+`pw-loopback -m '{{[FL FR]}}' --capture-props='{{media.class=Audio/Sink node.name=my-sink}}'`
+
+- Create a sink that can be selected in applications and that swaps the left and right channels to another sink:
+
+`pw-loopback --capture-props='media.class=Audio/Sink node.name=my-sink audio.position=[FL FR]' --playback-props='audio.position=[FR FL]'`
```
Comment (navarroaxel) [inline pages/linux/pw-loopback.md:16]:
```suggestion
`pw-loopback --capture-props='{{media.class=Audio/Sink node.name=my-sink audio.position=[FL FR]}}' --playback-props='{{audio.position=[FR FL]}}'`
```

File: pages/linux/pw-loopback.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Create a source that can be selected in applications and that swaps the left and right channels from another source:
+
+`pw-loopback --capture-props='audio.position=[FR FL]' --playback-props='media.class=Audio/Source node.name={{my-source}} audio.position=[FL FR]'`
```
Comment (navarroaxel) [inline pages/linux/pw-loopback.md:20]:
```suggestion
`pw-loopback --capture-props='{{audio.position=[FR FL]}}' --playback-props='{{media.class=Audio/Source node.name=my-source audio.position=[FL FR]}}'`
```

File: pages/linux/pw-loopback.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# pw-loopback
+
+> Tool for creating loopback devices in pipewire.
+> More information: <https://gitlab.freedesktop.org/pipewire/pipewire/-/wikis/Virtual-Devices>.
+
+- Create a loopback device with the default loopback behavior:
+
+`pw-loopback`
+
+- Create a sink that can be selected in applications and that forwards the data to another sink:
```
Comment (sbrl) [inline pages/linux/pw-loopback.md:10]: Rogue `and` here?
```suggestion
- Create a sink that can be selected in applications that forwards the data to another sink:
```

Same for the other examples.

File: pages/linux/pw-loopback.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`pw-loopback -m '{{[FL FR]}}' --capture-props='{{media.class=Audio/Sink node.name=my-sink}}'`
+
+- Create a sink that can be selected in applications and that swaps the left and right channels to another sink:
```
Comment (sbrl) [inline pages/linux/pw-loopback.md:14]:
```suggestion
- Create a sink that can be selected in applications that swaps the left and right channels to another sink:
```

File: pages/linux/pw-loopback.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`pw-loopback --capture-props='{{media.class=Audio/Sink node.name=my-sink audio.position=[FL FR]}}' --playback-props='{{audio.position=[FR FL]}}'`
+
+- Create a source that can be selected in applications and that swaps the left and right channels from another source:
```
Comment (sbrl) [inline pages/linux/pw-loopback.md:18]:
```suggestion
- Create a source that can be selected in applications that swaps the left and right channels from another source:
```

**Review by sbrl — APPROVED** [review]
Thanks, @Managor!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**